Userland/Libraries/LibWeb/Cookie/Cookie.h

```diff
@@ -12,6 +12,13 @@
 
 namespace Web::Cookie {
 
+enum class SameSiteMode {
```

Comment:
I think this can just be called `SameSite`, same for the `same_site_mode_to_string` function. The member name is already using that name without "mode", and mixing names just creates confusion.